test/crab_HAA4bAnalysis.py

```diff
@@ -16,9 +16,9 @@
 config.Data.inputDBS = 'phys03'
 config.Data.splitting = 'FileBased'
 config.Data.unitsPerJob = 5
-config.Data.outLFNDirBase = '/store/user/%s/' % (getUsernameFromSiteDB())
+config.Data.outLFNDirBase = '/store/user/aashah/'
```

Comment:
Please revert to the previous line. getUsernameFromSiteDB automatically finds your username, so we can both use it at the same time